static/app/gettingStartedDocs/javascript/remix.tsx

```diff
@@ -0,0 +1,244 @@
+replaysOnErrorSampleRate: 1.0, // If you're not already sampling the entire session, change the sample rate to 100% when sampling sessions where errors occur.
+`;
+
+const performanceIntegration = `
+new Sentry.BrowserTracing({
+  // Set 'tracePropagationTargets' to control for which URLs distributed tracing should be enabled
+  tracePropagationTargets: ["localhost", "https:yourserver.io/api/"],
+  routingInstrumentation: Sentry.remixRouterInstrumentation(
+    useEffect,
+    useLocation,
+    useMatches
+  ),
+}),
+`;
+
+const performanceOtherConfig = `
+// Performance Monitoring
+tracesSampleRate: 1.0, // Capture 100% of the transactions, reduce in production!
+`;
+
+const prismaConfig = `
+new Sentry.Integrations.Prisma({ client: prisma })
+`;
+
+export const steps = ({
+  sentryInitContent,
+  sentryInitContentServer,
+}: {
+  sentryInitContent?: string;
+  sentryInitContentServer?: string[];
+} = {}): LayoutProps['steps'] => [
+  {
+    language: 'bash',
+    type: StepType.INSTALL,
+    configurations: [
+      {
+        description: t(
+          'Sentry captures data by using an SDK within your application’s runtime.'
+        ),
+        code: `
+        # Using yarn
+        yarn add @sentry/remix
+
+        # Using npm
+        npm install --save @sentry/remix
+        `,
+      },
+    ],
+  },
+  {
+    language: 'javascript',
+    type: StepType.CONFIGURE,
+    configurations: [
+      {
+        description: t(
+          'Import and initialize Sentry in your Remix entry points for both the client and server:'
+        ),
+        code: `
+        import { useLocation, useMatches } from "@remix-run/react";
+        import * as Sentry from "@sentry/remix";
+        import { useEffect } from "react";
+
+        Sentry.init({
+          ${sentryInitContent}
+        });
+        `,
+      },
+      {
+        description: tct(
+          `Initialize Sentry in your entry point for the server to capture exceptions and get performance metrics for your [action] and [loader] functions. You can also initialize Sentry's database integrations, such as Prisma, to get spans for your database calls:`,
+          {
+            action: (
+              <ExternalLink href="https://remix.run/docs/en/v1/api/conventions#action" />
+            ),
+            loader: (
+              <ExternalLink href="https://remix.run/docs/en/1.18.1/api/conventions#loader" />
+            ),
+          }
+        ),
+        code: `
+
+        ${
+          (sentryInitContentServer ?? []).length > 1
+            ? `import { prisma } from "~/db.server";`
+            : ''
+        }
+
+        import * as Sentry from "@sentry/remix";
+
+        Sentry.init({
+          ${sentryInitContentServer?.join('\n')}
+        });
+        `,
+      },
+      {
+        description: t(
+          'Lastly, wrap your Remix root with "withSentry" to catch React component errors and to get parameterized router transactions:'
+        ),
+        code: `
+import {
```

Comment:
I know this looks odd but without this strange format, the code snippet does not look nice when rendered. I will have to check how we can better write code snippets as string